snackpub/src/index.ts

```diff
@@ -40,23 +47,65 @@ function closeServerAsync(server: Server): Promise<void> {
   });
 }
 
+function terminateSocket(
+  socket: Socket<ClientToServerEvents, ServerToClientEvents, InterServerEvents, SocketData>,
+  reason: string
+) {
+  socket.emit('terminate', reason);
+  setTimeout(() => {
+    if (socket.connected) {
+      socket.disconnect(true);
+    }
+  }, 3000);
```

Comment:
Does this 3s timeout has a specific meaning here? If so, might be nice to add a comment on why 3s was chosen

Reply:
added a comment

```
  // When clients receive the `terminate` message, they should disconnect and no longer try to reconnect back to the server.
  // In case the clients do not handle the `terminate` message, e.g. from other websocket clients other than snackpub clients.
  // We will wait three seconds to close the connections actively.
```